balances-adjustment/src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationImpl.java

```diff
@@ -74,27 +85,34 @@ public CompletableFuture<BalanceComputationResult> run(Network network, String w
         network.getVariantManager().setWorkingVariant(workingVariantCopyId);
 
         do {
+            Reporter iterationReporter = Reports.createBalanceComputationIterationReporter(reporter, context.getIterationNum());
             // Step 1: Perform the scaling
+            Reporter scalingReporter = iterationReporter.createSubReporter("scaling", "Scaling");
             context.getBalanceOffsets().forEach((area, offset) -> {
                 Scalable scalable = area.getScalable();
                 double done = scalable.scale(network, offset, parameters.getScalingParameters());
+                Reports.reportScaling(scalingReporter, area.getName(), offset, done);
                 LOGGER.info("Iteration={}, Scaling for area {}: offset={}, done={}", context.getIterationNum(), area.getName(), offset, done);
             });
 
             // Step 2: compute Load Flow
-            LoadFlowResult loadFlowResult = loadFlowRunner.run(network, workingVariantCopyId, computationManager, parameters.getLoadFlowParameters());
+            Reporter lfReporter = iterationReporter.createSubReporter("loadflow", "Load Flow");
```

Comment:
Just for coherence, in OLF we have key "loadFlow", message "Load flow ..."

Reply:
Done